utils/clients/tenderly.ts

```diff
@@ -99,8 +101,22 @@ async function simulateNew(config: SimulationConfigNew): Promise<SimulationResul
   // Set various simulation parameters
   const from = DEFAULT_FROM
   const value = values.reduce((sum, cur) => BigNumber.from(sum).add(cur), BigNumber.from(0)).toString()
-  const simBlock = proposal.endBlock.add(1)
-  const simTimestamp = BigNumber.from(latestBlock.timestamp).add(simBlock.sub(proposal.endBlock).mul(12))
+
+  // For Bravo governors, we use the block right after the proposal ends, and for OZ
+  // governors we arbitrarily use the next block number.
```

Comment:
for OZ, why do we use next block number? what if the proposal is still ongoing at that block? does it not prevent us from executing the simulation?

Reply:
Ah nice, yea we can simplify this as I just did in 44d8d5e511. I think this conditional was leftover from debugging something